GitInfo.txt

```diff
@@ -1 +1 @@
-9.0.0-ci.net9
+9.0.0-preview.1
```

Comment:
need to revert this back

Reply:
not sure what you mean... we are making net9.0 become pre1?